e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -107,14 +107,111 @@
               "createOptions": ""
             }
           },
+          "directMethodSender1": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "InvocationSource": {
+                "value": "Cloud"
+              },
```

Comment:
Module to module DM should be also included in connectivity tests.

Reply:
I created a task to add this and test: https://msazure.visualstudio.com/One/_workitems/edit/5972370